Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -26,45 +26,43 @@ public static class TestUtilities
         public static Scene primaryTestScene;
         public static Scene[] additiveTestScenes = new Scene[0];
 
-        public static void InitializeMixedRealityToolkit()
-        {
-            MixedRealityToolkit.ConfirmInitialized();
-        }
-
         /// <summary>
-        /// Destroys all scene assets that were created over the course of testing
+        /// Destroys all scene assets that were created over the course of testing.
+        /// Used only in editor tests.
         /// </summary>
-        public static void TearDownScenes()
+        public static void EditorTearDownScenes()
         {
 #if UNITY_EDITOR
-            // If any of our scenes were saved, tear down the assets
-            SceneAsset primaryTestSceneAsset = AssetDatabase.LoadAssetAtPath<SceneAsset>(primaryTestSceneTemporarySavePath);
-            if (primaryTestSceneAsset != null)
+            if (!EditorApplication.isPlaying)
             {
-                AssetDatabase.DeleteAsset(primaryTestSceneTemporarySavePath);
-            }
+                // If any of our scenes were saved, tear down the assets
+                SceneAsset primaryTestSceneAsset = AssetDatabase.LoadAssetAtPath<SceneAsset>(primaryTestSceneTemporarySavePath);
+                if (primaryTestSceneAsset != null)
```

Comment:
I'm curious, why do we need to delete scenes we created in edit mode, but not onces created during playmode tests?